### ISSUE TITLE
SRT Listenter and MPEG-TS UDP output

### ISSUE THREAD

**Issue by mrecords121** [ALLATRA-IT/ephyr#51]
Could you add not only SRT Caller output but also SRT Listerner and MPEG-TS UDP over FFMPEG?

Best regards,
Maksym

**Comment by skhalymon** [comment]
Hi @mrecords121!

Are you asking about SRT Listener as input?

**Comment by mrecords121** [comment]
> Hi @mrecords121!
> 
> Are you asking about SRT Listener as input?

No, I mean SRT output in Listener mode (like RTMP Pull). You can find information about this mode in “SRT Alliance Deployment Guide”. By the way, you have implemented SRT Listerner (not just input but also the ouput) in you ALLATRA-IT/srt-server repository.

**Comment by mrecords121** [comment]
I'm not quite good at programming, but it seems to me that the implementation of MPEG-TS UDP in ffmpeg is completely similar to SRT.  Perhaps you will be able also to inject MPEG-TS UDP output into your server. (RTMP Push input to MPEG-TS UDP output)
